tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -72,6 +72,13 @@ jobs:
       workingFolder: '$(Build.BinariesDirectory)\$(BuildConfig)'
       createLogFile: true
 
+  - task: PublishSymbols@2
+    displayName: 'Publish Build Symbols'
+    inputs:
+      symbolsFolder: $(Build.BinariesDirectory)\$(BuildConfig)
+      searchPattern: '**/*.pdb'
```

Comment:
Probably you only need the onnxruntime.pdb.

Reply:
Also WinML/DML, and possibly more stuff that might get built or added in the future. I thought it was better to upload everything.

Reply:
PDBs are huge. We may run out of storage quickly.

Reply:
"symbolsFolder: $(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig)" would be better.